src/main/java/org/jitsi/protocol/xmpp/AbstractOperationSetJingle.java

```diff
@@ -154,6 +159,7 @@ public boolean replaceTransport(
         }
 
         IQ reply = getConnection().sendPacketAndGetReply(jingleIQ);
+        stats.stanzaSent(jingleIQ.getAction());
```

Comment:
Not that I understand jicofo, but `reply` can apparently be `null` here -- does that imply that a stanza was not in fact sent?

Reply:
More likely that a stanza was sent, but no response was received. I think "number of stanzas we tried to send" is a reasonable meaning for the counter.